frontend/app/forms/proof-of-completed-service/proof-of-completed-service.web.component.js

```diff
@@ -113,40 +113,40 @@ export default function ProofOfCompletedService_Web({ data }) {
           listed in section 3 by mail or commerical courier service to:
         </p>
         <TextInput
-          dataKey="proofofcompletedservice_person.fullname"
+          dataKey="person.proofOfCompletedServiceMailFullname"
```

Comment:
Why is that (and the other addressee fields) `person. ...` and not `case. ...`? I'm still learning what the conventions are, and wanted to know why you chose it.

Reply:
I think in this case, we need the person to be specific to this form. This part of the form is asking who the documents were mailed to. These fields should not be autopopulated with the same name as the plaintiff/petitioner. With this in mind, would the way I coded it be appropriate or would modification still be required?

Reply:
This information is specific to the case. See https://www.utcourts.gov/howto/service/docs/1020GE_Proof_of_Service.pdf where it asks for one case number for this information

<img width="429" alt="image" src="https://user-images.githubusercontent.com/5524384/69014993-e73e3680-094c-11ea-9c16-e6231db10a6e.png">

Reply:
Corrected the code to use `case` datakey.